aptos.go

```
@@ -84,6 +93,11 @@ func AptosNameFromChainId(chainId uint64) (string, error) {
 func AptosChainIdFromSelector(selector uint64) (uint64, error) {
 	chain, exist := aptosChainsBySelector[selector]
 	if !exist {
+		// Try remote datasource if enabled (selectors are globally unique)
+		if _, chainID, _, ok := getRemoteChainBySelector(selector); ok {
+			id, _ := strconv.ParseUint(chainID, 10, 64)
```

can we confidently ignore the errors? I fear we might get bitten by a bad manual edit of the all_selectors.yml file.